mmv1/third_party/terraform/resources/resource_storage_transfer_job.go

```diff
@@ -82,6 +82,19 @@ func resourceStorageTransferJob() *schema.Resource {
 					Schema: map[string]*schema.Schema{
 						"object_conditions": objectConditionsSchema(),
 						"transfer_options":  transferOptionsSchema(),
+						"source_agent_pool_name": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Computed:    true,
+							Description: `Specifies the agent pool name associated with the posix data source. When unspecified, the default name is used.`,
+						},
+						"sink_agent_pool_name": {
```

Comment:
Looks like we have `ForceNew` added only for `sink_agent_pool_name`, does it mean `source_agent_pool_name` supports in-place update but `sink_agent_pool_name` does not? If so, we can remove the update step for `sink_agent_pool_name` test, since it's the same as testing creation + deletion of the resource (`ForceNew` means when user is attempting to update the field, Terraform will delete and recreate the resource).

Reply:
Attempting to update the "sink_agent_pool_name" and "source_agent_pool_name" points to an existing agent pool or creates a new agent pool, but it doesn't delete any existing agent pool.  I believe, we can remove the update altogether for both the fields as it will be the same creation+deletion of the resource, What are your thoughts?